apps/dialtone-documentation/docs/.vuepress/theme/components/DialtoneLogo.vue

```diff
@@ -1,14 +1,15 @@
 <template>
   <router-link
-    class="d-btn h:d-bgc-transparent d-py6 d-w216 d-jc-flex-start"
+    class="d-btn d-btn--muted h:d-bgc-transparent d-py6 d-px12 d-w216 d-jc-flex-start"
```

Comment:
Probably shouldn't have been a `d-btn` to begin with. 
```suggestion
class="d-pl8 d-w216 d-fc-secondary d-td-none"
```